deploy/kustomize/storageclass/pmem-storageclass.yaml

```diff
@@ -3,12 +3,3 @@ kind: StorageClass
 metadata:
   name: pmem-csi-sc
 provisioner: pmem-csi.intel.com
-# Setting allowedTopology ensures that pods using volumes
-# with late binding only get scheduled to pmem
-allowedTopologies:
```

Comment:
@pohly Why should we remove `allowedTopologies` form the storageclass?

Reply:
Because it didn't work - see the commit message in https://github.com/intel/pmem-csi/pull/838/commits/7c4ee8639eb7729a6ea455a1d1dcbca0af26d5de

Reply:
If I am not wrong That attempt was for using `{"pmem-csi.intel.com/driver":"started"}` as allowed topology. Isn't it?. But it should work for `{storage: "pmem}` as before. And anyways your are passing the same label for controller's `--nodeSelecter` [here](https://github.com/intel/pmem-csi/pull/838/commits/3a7d2cfc9483ad24c52c83a833865c02e158ea35?file-filters%5B%5D=.csv&file-filters%5B%5D=.go&file-filters%5B%5D=.make&file-filters%5B%5D=.sh&file-filters%5B%5D=.yaml#diff-c4a858c419d0c78037540b85e6cb044a293f53c45c5a073e68108ed890430ea6R183).  Then why exactly this `--nodeSelector` is needed as we get the same info from from the storage class?

Reply:
> If I am not wrong That attempt was for using `{"pmem-csi.intel.com/driver":"started"}` as allowed topology.

That's the first attempt (https://github.com/intel/pmem-csi/pull/838/commits/a03a7004e8288fca31b13a752019399328f37256). The second attempt was `{storage: pmem}` which also didn't work.

> Then why exactly this --nodeSelector is needed as we get the same info from from the storage class?

We can't use `AllowedTopology` at all at the moment, not with the currently released external-provisioner. Even if we can start using `AllowedTopology` with some future external-provisioner, then the label check in kube-scheduler may still go wrong when labels get changed (less likely, but can happen). The rescheduler then handles that case and thus remains useful.